lib/client/client.go

```diff
@@ -55,6 +54,23 @@ func New(config api.CalicoAPIConfig) (*Client, error) {
 	return &cc, err
 }
 
+// NewFromEnv loads the config from ENV variables and returns a connected Client.
+func NewFromEnv() (*Client, error) {
+
+	config, err := LoadClientConfigFromEnvironment()
+	if err != nil {
+		return nil, err
+	}
+
+	cc := Client{}
```

Comment:
Good point! I had more logic there at one point, but reverted back and forgot to change it to `New()`